### PR TITLE
add covid-19 demo and readme

### PR DESCRIPTION
**Pull request by roywei**
Add Covid19 detection demo
Update Readme

### CODE REVIEW

File: covid19-detection/build.gradle [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+repositories {
+    mavenCentral()
+    maven {
+        url "https://oss.sonatype.org/content/repositories/snapshots/"
+    }
+}
+
+dependencies {
+    implementation "ai.djl:api:0.5.0-SNAPSHOT"
```
Comment (zachgk) [inline covid19-detection/build.gradle:19]: Can this target 0.4.x so it doesn't break with a later version release?

Comment (frankfliu) [reply covid19-detection/build.gradle:19]: We don't have 0.4.x release for tensorflow api and native jar

File: covid19-detection/README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+This is an example to demonstrate how to use [Deep Java Library](djl.ai) to detect COVID-19 based on X-ray images.
+
+For more details, please follow this [blog post]()
+
+## Setup
+run `./gradlew build`
+
+## Run prediction on images
+run `./gradlew run -Dmain=com.examples.Covid19Detection --args="/path/to/saved/model /path/to/image`
```
Comment (frankfliu) [inline covid19-detection/README.md:11]: 1. -Dmain= .. is not necessary
2. use -Dai.djl.repository.zoo.location=/path/to/saved/model

File: covid19-detection/build.gradle [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    implementation "ai.djl:repository:0.5.0-SNAPSHOT"
+    implementation "ai.djl.tensorflow:tensorflow-api:0.5.0-SNAPSHOT"
+    implementation "ai.djl.tensorflow:tensorflow-engine:0.5.0-SNAPSHOT"
+    implementation "ai.djl.tensorflow:tensorflow-model-zoo:0.5.0-SNAPSHOT"
+    implementation files("libs/libtensorflow.jar")
```
Comment (frankfliu) [inline covid19-detection/build.gradle:24]: implemenation "ai.djl.tensorflow:tensorflow-native-auto:0.5.0-SNAPSHOT"

File: covid19-detection/build.gradle [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+application {
+    mainClassName = System.getProperty("main", "com.examples.Covid19Detection")
+}
+
+configure(this) {
+    defaultTasks "jar"
+
+    apply from: file("${rootProject.projectDir}/../tools/gradle/formatter.gradle")
+}
```
Comment (frankfliu) [inline covid19-detection/build.gradle:39]: tasks.distTar.enabled = false

File: covid19-detection/src/main/java/com/examples/Covid19Detection.java [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+import java.util.Arrays;
+import java.util.List;
+
+public class Covid19Detection {
+
+    @SuppressWarnings("PMD.SystemPrintln")
+    public static void main(String[] args)
+            throws IOException, MalformedModelException, TranslateException {
+        String modelPath = args[0];
+        String imagePath = args[1];
```
Comment (frankfliu) [inline covid19-detection/src/main/java/com/examples/Covid19Detection.java:39]: Use Criteria with local model zoo to load model

**Pull request merged**